### ISSUE TITLE
Bug in site settings

### ISSUE THREAD

**Issue by leigh-pointer** [oqtane/oqtane.framework#2128]
When logged in as Admin  - change nothing - click save -  message "You Must Provide A Site Name, Alias, And Default Theme/Container"

![image](https://user-images.githubusercontent.com/9447612/163175181-64c9eeef-3b28-4c0d-8545-b790ff3fb4f1.png)

**Comment by sbwalker** [comment]
@leigh-pointer I am guessing there is something wrong with your Alias specification? Has the Default Alias been specified?

**Comment by leigh-pointer** [comment]
@sbwalker yes the default alias is set.  I have no issue when logged in as host only when admin.

**Comment by sbwalker** [comment]
Thank you... that is a very helpful hint

**Comment by leigh-pointer** [comment]
@sbwalker The issue is that the urls are only loaded if the logged in user is a host. The admin is not allowed to assign an Alias so will fail

**Comment by sbwalker** [comment]
this is fixed...
